src/zano/zanoTypes.ts

```diff
@@ -1,11 +1,13 @@
 import {
+  asArray,
   asCodec,
   asNumber,
   asObject,
   asOptional,
   asString,
   Cleaner
 } from 'cleaners'
+import { TransferParams } from 'react-native-zano'
```

Comment:
`import type`, just to be extra safe. If we import any actual code, it'll blow up the build.